### PR TITLE
Bartworks cleaning

### PR DESCRIPTION
**Pull request by Connor-Colenso**
Removed some old stuff, cleaned some areas up.

- Fixed some Germanglish.
- Cleaned up the borosilicate API.
- Reduce the amount of valid glass types to a core sensible selection. No more bizarre biovat monstrosities.
- Removed some asm nonsense that was changing the mod container. Not needed.
- Generally removed some unused methods and classes.
- Removed "added by bartworks" from every single item. Still on some machines.
- Cleaned up the energy distributor and diode tooltips. They were double showing information before.
- Remove magnetos, dead content. 
- Remove destructopack. Redundant with trashcan in inventoru from SU. (Needs a quest PR)

Should be all. Nothing particularly major. Tested in full pack, but will need a small change in NEI diagrams.

### CODE REVIEW

**Comment by Dream-Master** [comment]
i am agiant scrap the magneto stuff. we can use then elsewhere. the need to removed from oredict for sure
destructor pack is used maybe in quests and need a change there too.

**Comment by Connor-Colenso** [comment]
> i am agiant scrap the magneto stuff. we can use then elsewhere. the need to removed from oredict for sure destructor pack is used maybe in quests and need a change there too.

Magnetos don't serve any purpose anymore, Bart added them 5 years ago. They are dead content. I will be making a PR regarding the destructopack thing, mentioned that in the description.

File: src/main/java/com/github/bartimaeusnek/bartworks/MainMod.java [hunk 1]
```diff
@@ -193,12 +177,12 @@ public void onServerStarting(FMLServerStartingEvent event) {
...
 
     @Mod.EventHandler
-    public void onServerStarted(FMLServerStartedEvent event) {
+    public void onServerStarted(FMLServerStartedEvent ignored) {
         MainMod.runOnPlayerJoined(ConfigHandler.classicMode, ConfigHandler.disableExtraGassesForEBF);
     }
 
     @Mod.EventHandler
-    public void onModLoadingComplete(FMLLoadCompleteEvent event) {
+    public void onModLoadingComplete(FMLLoadCompleteEvent ignored) {
```
Comment (miozune) [inline src/main/java/com/github/bartimaeusnek/bartworks/MainMod.java:185]: nit: I think it's common thing to leave event variable name just as is even if it's not used. If you're annoyed by Intellij warning, you can configure "suppress unused warning if annotated by."

Comment (Connor-Colenso) [reply src/main/java/com/github/bartimaeusnek/bartworks/MainMod.java:185]: It doesn't seem to make any difference, is there any particular reason?

Comment (miozune) [reply src/main/java/com/github/bartimaeusnek/bartworks/MainMod.java:185]: It's just common to not mark it as ignored among almost all the mods including future versions. Plus if someone decides to make use of the variable, it being labeled as ignored is... a bit weird, even if you can just change it back.

**Review by boubou19 — APPROVED** [review]

**Pull request merged**